Webapp/sources/services/workgroup.py

```diff
@@ -122,24 +106,28 @@ def get_workgroup_sm(workgroup_name: str) -> List[models.User]:
     )
 
 
-def get_user_type_in_workbook(workgroup_name: str) -> str:
+def get_user_type(workgroup_name: str, user: models.User) -> str:
     """
     Returns the user type of the current user in the specified workgroup
     Args:
         workgroup_name: str, name of workgroup to search
+        user: models.User, the user for which to check membership
 
     Returns:
-        user_type: str, user membership type in specified workbook
+        user_type: str, user membership type in specified workgroup or None if user is not in workgroup
     """
+    user_type = None
+
     pi = get_workgroup_pi(workgroup_name)
-    if current_user.email in [user.email for user in pi]:
+    if user.email in [user.email for user in pi]:
         user_type = "principal_investigator"
```

Comment:
Not something to fix in this PR, but I would consider at some point turning these magic strings like `"principal_investigator"` etc. into a `StrEnum` or just an `Enum` if your python version is less than 3.11. That way if you're using these all throughout the app, you would only have to change the enum and it would be applied everywhere

Reply:
Thanks, I'll definitely do that!